etq:q

```diff
@@ -1,251 +0,0 @@
-[33mcommit a3aca1357b6126d776d7706fdfbc9196084534a6[m[33m ([m[1;36mHEAD -> [m[1;32mretrieve-data-in-one-place[m[33m)[m
```

Comment:
what is the file doing here?

Reply:
I deleted it - not sure how it got in there. Looking at the commit history looks like I somehow generated it and commited it and push it up accidentally. Looking at the file name, looks like I was trying to enter a vim command somewhere but I wasn't typing it in the right place